packages/uniforms-bootstrap4/src/components/fields/ListAddField.js

```diff
@@ -24,4 +24,8 @@ const ListAdd = ({
     );
 };
 
+ListAdd.defaultProps = {
+    addIcon: (<i className="octicon octicon-plus" />),
```

Comment:
Sorry for being so picky _(just say no, if you want)_, but could you remove the parentheses and trailing comma?